cli/pkg/kubectl/bind/plugin/bind.go

```diff
@@ -159,7 +159,7 @@ func (b *BindOptions) Run(ctx context.Context, urlCh chan<- string) error {
 	}
 
 	if provider.APIVersion != kubebindv1alpha2.GroupVersion {
-		return fmt.Errorf("unsupported binding provider version: %q", provider.APIVersion)
+		return fmt.Errorf("unsupported binding provider version: %q != %q", provider.APIVersion, kubebindv1alpha2.GroupVersion)
```

Comment:
Nit: rephrase to make it clear which is the actual and which is the expected value:

> "unsupported binding provider version %q, expected %q"